chronograf.go

```diff
@@ -143,6 +187,13 @@ type DashboardQuery struct {
 	QueryConfig QueryConfig `json:"queryConfig,omitempty"` // QueryConfig represents the query state that is understood by the data explorer
 }
 
+// TemplateQuery is used to retrieve choices for template replacement
+type TemplateQuery struct {
+	Command string `json:"query"`        // Command is the query itself
+	DB      string `json:"db,omitempty"` // DB is optional and if empty will not be used.
+	RP      string `json:"rp,omitempty"` // RP is a retention policy and optional; if empty will not be used.
```

Comment:
Hey @121watts , yup, I have that already here: https://github.com/influxdata/chronograf/blob/df6b90cc03b8cd1d84b6ad2ebab73390aaac4887/chronograf.go#L129-L130